tests/e2e/api/general.test.ts

```diff
@@ -127,17 +131,19 @@ describe("general api", () => {
       const config = new AptosConfig({ network: Network.LOCAL });
       const aptos = new Aptos(config);
 
-      const payload: InputViewRequestData = {
+      const payload: InputViewFunctionData = {
         function: "0x1::account::exists_at",
+        typeArguments: [],
```

Comment:
this is optional right? lets not add it if not using it, is confuses end users. We could have a test with type arguments values

Reply:
Yeah, it can be optional